proxy_multi_map.go

```diff
@@ -178,6 +178,21 @@ func (m *MultiMap) ContainsEntry(ctx context.Context, key interface{}, value int
 	return codec.DecodeMultiMapContainsEntryResponse(response), nil
 }
 
+// ValueCount returns the number of values that match the given key in the multi-map.
+func (m *MultiMap) ValueCount(ctx context.Context, key interface{}) (int, error) {
```

Comment:
Can you refactor according to this practice?
https://dave.cheney.net/practical-go/presentations/qcon-china.html#_return_early_rather_than_nesting_deeply

In summary lets try not to use nested if/else blocks, instead return early incase of an error

Reply:
Thanks, resolved: [9ab0e17](https://github.com/hazelcast/hazelcast-go-client/pull/734/commits/9ab0e1786fcc3f4474facea567467d9e521cc78b)

Reply:
Thanks!